cdap-docs/admin-manual/source/hadoop-compatibility.rst

```diff
@@ -47,8 +47,8 @@ CDAP currently requires Internet access on the CDAP service nodes (or until the
 
 For a Distributed CDAP cluster, version |release|, you **must** install these Hadoop components:
 
-.. |cdh-version|       replace:: 5.0.0 through 5.7.0
-.. |cdh-version-hbase| replace:: 5.0.0 through 5.7.0 (1.2-cdh5.7.0)
+.. |cdh-version|       replace:: 5.1 through 5.8
+.. |cdh-version-hbase| replace:: 5.1 through 5.8 (1.2-cdh5.8.x)
```

Comment:
OK, I'm stumped. Not sure what you meant; I was just bringing in @dereklwood updates to the Hadoop list. Should we be saying "1.2-cdh5.8.0" instead of "1.2-cdh5.8.x"?

Derek had asked why we were specifying point releases for Cloudera (and not for others) so I switched to the ".x", but if that is important, let's go back.